diff -up arts-1.5.8/artsc/artsdsp.c.glibc arts-1.5.8/artsc/artsdsp.c
--- arts-1.5.8/artsc/artsdsp.c.glibc	2007-10-08 04:47:09.000000000 -0500
+++ arts-1.5.8/artsc/artsdsp.c	2007-10-12 11:44:41.000000000 -0500
@@ -203,7 +206,7 @@ static int is_sound_device(const char *p
   return 0;
 }
 
-int open (const char *pathname, int flags, ...)
+int (open) (const char *pathname, int flags, ...)
 {
   va_list args;
   mode_t mode = 0;
@@ -594,7 +597,7 @@ ssize_t write (int fd, const void *buf, 
   return 0;
 }
 
-ssize_t read (int fd, void *buf, size_t count)
+ssize_t (read) (int fd, void *buf, size_t count)
 {
   CHECK_INIT();
 
diff -up arts-1.5.8/artsc/stdioemu.c.glibc arts-1.5.8/artsc/stdioemu.c
--- arts-1.5.8/artsc/stdioemu.c.glibc	2005-09-10 03:13:34.000000000 -0500
+++ arts-1.5.8/artsc/stdioemu.c	2007-10-12 11:45:05.000000000 -0500
@@ -83,9 +86,9 @@ static FILE *fake_fopen(const char *path
 		if(*mptr == '+') open_mode |= 3; /* 3 = readwrite */
 		if(*mptr == 'a') open_mode |= 2; /* append -> write */
   	}
-  	if(open_mode == 1) fdc->fd = open(path,O_RDONLY,0666);
-  	if(open_mode == 2) fdc->fd = open(path,O_WRONLY,0666);
-  	if(open_mode == 3) fdc->fd = open(path,O_RDWR,0666);
+  	if(open_mode == 1) fdc->fd = (open)(path,O_RDONLY,0666);
+  	if(open_mode == 2) fdc->fd = (open)(path,O_WRONLY,0666);
+  	if(open_mode == 3) fdc->fd = (open)(path,O_RDWR,0666);
 
 	if(open_mode && fdc->fd > 0)
 	{
